src/index.js

```diff
@@ -45,7 +47,16 @@ const App = () => {
         setToDos([...todos, newTodoItem]);
 
         setTodoInputValue('');
-        setDateInputValue('');
+        setDateInputValue('');  
+    }
+
+    const validateDueDate = (typedDateInput) => {
```

Comment:
In order to validate the typed date, I need to intercept the setting of its state by adding this func `validateDueDate`